src/common/UnkBattleFunc5.inc.c

```diff
@@ -0,0 +1,15 @@
+#include "common.h"
+
+ApiStatus N(UnkBattleFunc5)(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    s32 actorID = get_variable(script, *args++);
+    s32 partID = get_variable(script, *args++);
+
+    if (actorID == -127) {
```

Comment:
If you look at the actor ID enum, I think this has a define for it.

Reply:
ACTOR_SELF